assets/scss/modules/_navigation.scss

```diff
@@ -147,3 +145,146 @@ a.back-to-top-link {
   }
 
 }
+
+/*
+Page Navigation
+
+A page navigation element with a previous and next link.
+
+Markup:
+<nav role="navigation" aria-label="Page navigation">
```

Comment:
> navigation role (semantics may be removed by using the presentation role)

That to me doesn't say that you don't need `role="navigation"`, but that it can be removed through the use of another attribute